support/values.yaml

```diff
@@ -71,3 +71,21 @@ grafana:
           access: proxy
           isDefault: true
           editable: false
+
+nfs-server-provisioner:
+  storageClass:
+    # Users will have to opt-in to using NFS-type storage under
+    # singleuser.storage.dynamic.storageClass
+    name: nfs
+    create: true
+    defaultClass: false
+    # Mount Options: default v3, should we use v4?
```

Comment:
We should definitely use v4! Lots of improvements in the protocol, leads to fewer 'stuck' clients

Reply:
Done in [`b714fdc`](https://github.com/2i2c-org/pilot-hubs/pull/613/commits/b714fdc74179a44eff9aec0a377764129d8477c7)